.repo-metadata.json

```diff
@@ -3,11 +3,12 @@
     "name_pretty": "Google Auth httplib2",
     "client_documentation": "",
     "issue_tracker": "https://github.com/googleapis/google-auth-library-python-httplib2/issues",
-    "release_level": "alpha",
+    "release_level": "preview",
     "language": "python",
     "library_type": "AUTH",
     "repo": "googleapis/google-auth-library-python-httplib2",
     "distribution_name": "google-auth-httplib2",
     "default_version": "",
-    "codeowner_team": ""
+    "codeowner_team": "",
+    "api_shortname": "google-auth-httplib2"
```

Comment:
This is not a GAPIC, is an `api_shortname` required?

Reply:
Fixed.